Gemfile

```diff
@@ -15,7 +15,7 @@ if ENV["DB"] == "mysql" || ENV["DB"] == "mariadb"
 end
 gem "pg", "~> 1.0" if ENV["DB"] == "postgresql"
 
-gem "alchemy_i18n", github: "AlchemyCMS/alchemy_i18n", branch: "download-flatpickr-locales"
+gem "alchemy_i18n", github: "AlchemyCMS/alchemy_i18n", branch: "main"
```

Comment:
This has already been fixed in latest main https://github.com/AlchemyCMS/alchemy_cms/blob/main/Gemfile#L18